Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -469,6 +469,7 @@
     - CutterMachineCircuitboard
     - StationAnchorCircuitboard
     - SalvageMagnetMachineCircuitboard
+    - OperatingTableCircuitboard # Shitmed Change
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)